test/adapters/active_record_test.rb

```diff
@@ -128,7 +128,6 @@ def setup
   end
 
   def teardown
-    DatabaseCleaner.clean
```

Comment:
This is necessary to get tests to run locally. DB cleaner seems to run automatically on newer versions with minitest